docs/Home.md

```diff
@@ -0,0 +1,87 @@
+<!--
+ Copyright 2019 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in
+ compliance with the License. You may obtain a copy of the License at
+
+        https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under the License
+ is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ implied. See the License for the specific language governing permissions and limitations under the
+ License.
+-->
+# Google Compute Engine Plugin
+The Google Compute Engine (GCE) Plugin allows you to use GCE virtual machines (VMs) with Jenkins to execute build tasks. GCE VMs provision quickly, are destroyed by Jenkins when idle, and offer Preemptible VMs that run at a much lower price than regular VMs.
```

Comment:
Can you split long lines like this into separate lines no more than 100 characters wide? See what we did on the GKE plugin [docs](https://github.com/jenkinsci/google-kubernetes-engine-plugin/blob/develop/docs/Home.md). A single newline is rendered as a space so it doesn't impact how it's rendered here.

Exception: code blocks

Reply:
Done